File: Practice/21/Python/twenty-oneTask.py
def bmi(weight: float, height: float) -> float:
    height /= 100
    return weight / (height ** 2)


def print_bmi(BMI: float) -> float:
    if BMI < 18.5:
        print("Underweight")
    elif BMI < 25.0:
        print("Normal")
    elif BMI < 30.0:
        print("Overheight")
    else:
        print("Obesity")


weight, height = map(float, input().split())
print_bmi(bmi(weight, height))

File: Practice/21/C++/Twenty-oneTask/Twenty-oneTask/Twenty-oneTask.cpp
#include <iostream>
using namespace std;

double BMI(double weight, double height)
{
	height = height / 100;
	return (weight / (height * height));
}
void printBMI(double BMI)
{
	if (BMI < 18.5)
		cout << "Underweight";
	else if (BMI < 25.0)
		cout << "Normal";
	else if (BMI < 30.0)
		cout << "Overweight";
	else
		cout << "Obesity";
	cout << endl;
}
int main()
{
	double weight, height;
	cin >> weight >> height;
	printBMI(BMI(weight, height));
}